### PR TITLE
MEN-4429: configuration deployment end-point

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

File: client/workflows/client.go [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+	"github.com/mendersoftware/go-lib-micro/identity"
+	"github.com/mendersoftware/go-lib-micro/requestid"
+	"github.com/mendersoftware/go-lib-micro/rest.utils"
+	"github.com/pkg/errors"
+)
+
+const (
+	HealthCheckURI = "/api/v1/health"
+	ResetEmailURI  = "/api/v1/workflow/send_password_reset_email"
```
Comment (kjaskiewiczz) [inline client/workflows/client.go]: do we need this in the deviceconfig?

File: docs/management_api.yml [hunk 2]
```diff
@@ -102,7 +102,45 @@ paths:
...
       responses:
         204:
-          description: Created
+          description: Success
+        400:
+          description: Bad Request.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
```
Comment (kjaskiewiczz) [inline docs/management_api.yml:111]: to me, it looks like the endpoint can also return 404 not found

**Review by kjaskiewiczz — APPROVED** [review]
only two comments; otherwise look good
I'm approving the PR, I think docs can be updated with the next PR

File: client/workflows/client.go (hunk 1 above)
Comment (tranchitella) [reply client/workflows/client.go]: ops, sorry, copy-and-paste from another package. Removed!

File: docs/management_api.yml (hunk 2 above)
Comment (tranchitella) [reply docs/management_api.yml]: right, added, thanks!

**Comment by tranchitella** [comment]
@kjaskiewiczz I believe I'll need your approval again

**Review by kjaskiewiczz — APPROVED** [review]

**Pull request merged**